### ISSUE TITLE
Add an analyzer for <img> elements missing alt attributes

### ISSUE THREAD

**Issue by groovecoder** [mdn/webalyzer#12]
To start checking for accessibility issues, add an analyzer to process HTML and look for <img> elements that are missing alt attributes.

**Comment by peterbe** [comment]
That would ideally require the commencement of the second plugin (aka. the accessibility app).

An interesting challenge is; how do you present that of the 13,455 documents collected, there are 399 cases of `<img>` tags without an alt? You'd really have to start analyzing and diff'ing these 399 occurances and only present those that are sufficiently different.
